### PR TITLE
Board: fix attribute views for tags

### PR DESCRIPTION
**Pull request by annano**
Signed-off-by: Anna No <anna.no@xored.com>

# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [ ] - Does it well tested?
* [x] - Tested for Chrome.
* [ ] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

**Review by BykhovDenis — APPROVED** [review]

File: packages/presentation/src/components/AttributeBarEditor.svelte [hunk 1]
```diff
@@ -41,14 +42,24 @@
...
+    }
+    if (presenterClass.category === 'attribute') {
       const typeClass = hierarchy.getClass(presenterClass.attrClass)
       const editorMixin = hierarchy.as(typeClass, view.mixin.AttributeEditor)
       editor = getResource(editorMixin.editor).catch((cause) => {
         console.error(`failed to find editor for ${_class} ${attribute} ${presenterClass.attrClass} cause: ${cause}`)
       })
     }
-    if (presenterClass?.attrClass !== undefined && presenterClass?.category === 'array') {
+    if (presenterClass.category === 'collection' && allowedCollections.findIndex(attr => attr === attribute?.name) >= 0) {
```
Comment (Dvinyanin) [inline packages/presentation/src/components/AttributeBarEditor.svelte]: Aren't the collections already filtered by the `ClassAttributeBar`?

File: packages/presentation/src/components/AttributeBarEditor.svelte [hunk 2]
```diff
@@ -41,14 +42,24 @@
     attribute: AnyAttribute,
     presenterClass?: { attrClass: Ref<Class<Doc>>; category: AttributeCategory }
   ): void {
-    if (presenterClass?.attrClass !== undefined && presenterClass?.category === 'attribute') {
+    if (presenterClass?.attrClass === undefined) {
+      return
+    }
+    if (presenterClass.category === 'attribute') {
```
Comment (Dvinyanin) [inline packages/presentation/src/components/AttributeBarEditor.svelte]: Maybe extract `mixin` type into a variable?

Comment (annano) [reply packages/presentation/src/components/AttributeBarEditor.svelte]: Updated

File: packages/presentation/src/components/AttributeBarEditor.svelte (hunk 1 above)
Comment (annano) [reply packages/presentation/src/components/AttributeBarEditor.svelte]: Removed

**Review by Dvinyanin — APPROVED** [review]

**Pull request merged**